modules/hub/src/main/actorApi.scala

```diff
@@ -145,6 +145,68 @@ case class LichessThread(
   notification: Boolean = false)
 }
 
+package notify {
```

Comment:
wow wow wow. There's way too much code in here.

`modules/hub` is about declaring immutable messages to be sent from a module to another. The actual models should live in their respective modules.